src/storage/block_compactor.cpp

```diff
@@ -78,8 +78,8 @@ bool BlockCompactor::EliminateGaps(CompactionGroup *cg) {
   for (auto &entry : cg->blocks_to_compact_) {
     RawBlock *block = entry.first;
     std::vector<uint32_t> &empty_slots = entry.second;
-    TERRIER_ASSERT(block->GetInsertHead() == layout.NumSlots(),
-                   "The block should be full to stop inserts from coming in");
+    //    TERRIER_ASSERT(block->GetInsertHead() == layout.NumSlots(),
```

Comment:
Same comment as above: I don't think commenting these lines out is a long-term solution to being able to use the block compaction for the checkpointing process in addition to being able to use it for the compaction process.